src/main/webapp/dev/js/main/feedbackSubmissionsEdit.js

```diff
@@ -479,24 +479,61 @@ function prepareConstSumQuestions() {
                 numResponses = parseInt(numResponses, 10);
 
                 $(`#constSumInfo-${qnNum}-${numResponses - 1}`).show();
+            } else {
+                $(`[id^="constSumInfo-${qnNum}-"]`).show();
+                $(`[id^="constSumSubmissionFormOptionFragment-${qnNum}"]`).addClass('padding-left-55px');
+                $(`[id^="constSumSubmissionFormOptionFragment-${qnNum}"]`).addClass('margin-top-15px');
```

Comment:
@wenmogu  The reason behind further indentation in options style questions is because those input boxes are parameters that are evaluated for each student separately. Those don't belong to `Evaluee/Recipient`. In the above view 11, 22, etc. are parameters that are evaluated for each recipient. You should have another heading above that listing the recipient name ( A, B, etc.). Not 
sure why it isn't there. Then the alignment will look appropriate like this:
![image](https://user-images.githubusercontent.com/11634699/40487483-331851a4-5f5c-11e8-87d0-4a7ae39d70c0.png)

If I put Grades, Sports, etc. under the same column as Student names it looks confusing